dist/docker-images/ziti-controller/compose.yml

```diff
@@ -34,6 +34,12 @@ services:
       ZITI_CTRL_ADVERTISED_PORT: ${ZITI_CTRL_ADVERTISED_PORT:-1280}                   # TCP port of the controller
       ZITI_PWD: ${ZITI_PWD:-}                                                         # password for the default admin user
 
+      # *** create or join a cluster ***
+      ZITI_BOOTSTRAP_CLUSTER: true     # create the first node in the cluster unless ZITI_CLUSTER_NODE_PKI is set
+      ZITI_CLUSTER_TRUST_DOMAIN: ${ZITI_CLUSTER_TRUST_DOMAIN:-}  # spiffe trust domain shared by all nodes
```

Comment:
i'd like to discuss before we push it out

Reply:
ready to discuss - will plan time for Thursday